client/GameComponents/MenuPane.jsx

```diff
@@ -33,10 +33,10 @@ class MenuPane extends React.Component {
 
         var buttons = _.map(this.props.buttons, button => {
             var option = (
-                <button key={button.command + buttonIndex.toString()} className='btn btn-primary'
-                    onClick={event => this.onButtonClick(event, button.command, button.arg, button.method)}
-                    onMouseOver={event => this.onMouseOver(event, button.card)} onMouseOut={event => this.onMouseOut(event, button.card)}
-                    disabled={button.disabled}>{button.text}</button>);
+                <button key={ button.command + buttonIndex.toString() } className='btn btn-primary'
```

Comment:
No :( and what's more vscode enforces the other way, which is why so much of the code has it that way, but i find it quite unreadable